### ISSUE TITLE
Orion fusion theme does not have "error" color for Invalid text-area state

### ISSUE THREAD

**Issue by JB016205** [cerner/terra-core#3427]
# Bug Report

## Description
For other themes, the --terra-form-textarea-error-border-color is a red either in shadow or color, but for the Orion fusion theme it is only greys[1]:

![MicrosoftTeams-image (47)](https://user-images.githubusercontent.com/3382775/116418764-0b6c9b00-a802-11eb-9d92-5c989e756e39.png)

This means that there is no attention called to the fusion theme when data is invalid, which is a poor user experience as well as an inconsistency with the component. 

1: https://github.com/cerner/terra-core/blob/main/packages/terra-form-textarea/src/orion-fusion-theme/Textarea.module.scss#L51

## Steps to Reproduce
1. Find a text-area that handles isInvalid.
2. Make sure you are in Orion Fusion Theme.
3. Trigger invalid state.

## Additional Context / Screenshots
Default theme:
![Screen Shot 2021-04-28 at 9 59 03 AM](https://user-images.githubusercontent.com/3382775/116426082-6bfed680-a808-11eb-9473-6a5e04a682ac.png)

Fusion theme:
![Screen Shot 2021-04-28 at 9 58 57 AM](https://user-images.githubusercontent.com/3382775/116426107-70c38a80-a808-11eb-8db4-8cba150a4703.png)

## Expected Behavior
The fusion theme should have some form of attention color around the text-area for an invalid state.

## Possible Solution
Change one of the two below to match (or be close to) the default theme:
* --terra-form-textarea-error-border-color
* --terra-form-textarea-error-box-shadow

## Environment
* Component Name and Version: ion-conditions-js rendered within Powerchart
* Browser Name and Version: N/A
* Node/npm Version: [e.g. Node 8/npm 5] N/A
* Webpack Version: N/A
* Operating System and version (desktop or mobile): N/A

## @ Mentions
N/A

**Comment by neilpfeiffer** [comment]
@JB016205 This is by design with the orion-fusion-theme and the desired user experience for that theme, since the intent of that theme is to match the existing design patterns of the Cerner MPage Fusion library, which also does not have error state indication on its input borders: http://mpages-fusion.cerner.com/#form-textbox

<img width="1020" alt="Screen Shot 2021-04-30 at 9 39 42 AM" src="https://user-images.githubusercontent.com/1355485/116711985-00983e80-a999-11eb-921d-0a98325f724d.png">

The reason for the matching of the older patterns is so that when multiple instances of forms are on a single page from a combination of both Terra and Fusion sources, they all will have a consistent user experience, rather than some having an elevated user experience with the error borders, and others without, creating now a danger or risk in that a user will not notice them to the same degree of importance as the others. 

If in the future the patterns are enhanced for MPage Fusion to elevate their user experience, we will address it in both the Terra theme and with the Fusion library at the same time.

**Comment by JB016205** [comment]
No concerns from me, feel free to close if there's nothing else to follow up on.

**Comment by neilpfeiffer** [comment]
Thanks @JB016205, happy to provide the rationale.
